### ISSUE TITLE
BatchNorm parameters

### ISSUE THREAD

**Issue by adamgayoso** [YosefLab/scVI#308]
Is there a reason we are not using default batch norm hyperparameters? Is this something that was hand-tuned?

https://github.com/YosefLab/scVI/blob/e27cb81812e7bff1b8dfa3624292c353dfbcda96/scvi/models/modules.py#L38

**Comment by jeff-regier** [comment]
@Edouard360 Do you happen to recall whether the default batch norm parameters worked?

@gabmis We may want hyperopt to search over these batchnorm parameters too if it turns out that using something other than the default is important.

**Comment by romain-lopez** [comment]
I think we set them to copy tensorflow standards and reproduce the TF version of scVI.

**Comment by gabmis** [comment]
also, should we tune `use_batch_norm`?

**Comment by jeff-regier** [comment]
That might be interesting

**Comment by adamgayoso** [comment]
Should we provide the users an interface to change the BatchNorm parameters?
